ribs/emitters/_gradient_operator_emitter.py

```diff
@@ -1,15 +1,40 @@
-"""Provides the GradientEmitter."""
+"""Provides the GradientOperatorEmitter."""
 
 import numpy as np
 
 from ribs._utils import check_1d_shape, check_batch_shape, validate_batch_args
 from ribs.emitters._emitter_base import EmitterBase
 
 
-class GradientEmitter(EmitterBase):
-    """Generates new solutions based on the gradient of the objective and measures.
+class GradientOperatorEmitter(EmitterBase):
+    """Generates solutions with a gradient arborescence, with coefficients
+    parameterized by a fixed Gaussian distribution.
 
-    _extended_summary_
+    This emitter originates in `Fontaine 2021
+    <https://arxiv.org/abs/2106.03894>`_. It leverages the gradient information
+    of the objective and measure functions, generating new solutions around a
+    *solution point* :math:`\\boldsymbol{\\theta}` using *gradient
```

Comment:
It's no longer a single solution point theta like it is in GradientArborescenceEmitter. In this emitter, we're performing a gradient arborescence around each of the individual theta_i's that were returned in ask_dqd.